app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -37,7 +35,7 @@ object GameState {
     private var nextId = 0
 
     // Constants
-    private const val BASE_URL = "http://158.101.160.143:42069"
+    private const val BASE_URL = "http://10.0.2.2:42069"
```

Comment:
wrong url

Reply:
Damn it, we need to do something about this